client/app/l10n/en.json

```diff
@@ -331,7 +331,7 @@
   "in megabytes": "in megabytes",
   "in percentage": "in percentage",
   "in seconds": "in seconds",
-  "In this step the answers to the following questions are either missing or invalid:": "In this step the answers to the following questions are either missing or invalid:",
+  "In this step the answers to the following questions are either missing or invalid:": "These questions have missing or invalid answers:",
```

Comment:
This correction is not correct; The shown errors relate only to the current step.

Reply:
The communication director favoured the correction, so it isnt just unclear to me.
New try:
"The following questions to in this step have either missing or invalid answers:"